recipe/meta.yaml

```
@@ -10,27 +10,28 @@ source:
   sha256: 2ba4d248f9567a27c146b9772ef5dc93bd9622317978455ffe91b259340d13d1
 
 build:
-  number: 1
+  number: 0
+  # missing libjpeg-turbo for s390x
```

```suggestion
  # missing jpeg for s390x
```
Still valid but it's `jpeg`
```
conda search jpeg --platform s390x -c sfe1ed40
Loading channels: done
No match found for: jpeg. Search: *jpeg*
```

---

`jpeg` exists on `linux-s390x`
```
conda search jpeg --platform linux-s390x
Loading channels: done
# Name                       Version           Build  Channel
jpeg                              9b      h2a837d6_2  repo/main
jpeg                              9d      h2a837d6_0  repo/main
jpeg                              9e      h2a837d6_0  repo/main
jpeg                              9e      hc33a586_1  repo/main
jpeg                              9e      hc33a586_2  repo/main
jpeg                              9e      hc33a586_3  repo/main
```

---

Actually as mentioned by Serhii, it's available.
```
conda search jpeg --platform linux-s390x
Loading channels: done
# Name                       Version           Build  Channel
jpeg                              9b      h2a837d6_2  pkgs/main
jpeg                              9d      h2a837d6_0  pkgs/main
jpeg                              9e      h2a837d6_0  pkgs/main
jpeg                              9e      hc33a586_1  pkgs/main
jpeg                              9e      hc33a586_2  pkgs/main
jpeg                              9e      hc33a586_3  pkgs/main
```
Lesson learned - `conda search` requires full platform name